package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "telemetryclient-team-services-extension",
+  "version": "1.0.8",
+  "description": "Track telemetry data for your Team Services extension",
+  "homepage": "https://github.com/ALM-Rangers/Application-Insights-Client",
```

Comment:
@mikaelkrief I updated both to https://github.com/ALM-Rangers/telemetryclient-team-services-extension after renaming the reoo. Should they not be the same?